lib/facts/aix/scope6_interfaces.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Facts
+  module Aix
+    class Scope6Interfaces
+      FACT_NAME = 'scope6_.*'
+      TYPE = :legacy
+
+      def call_the_resolver
+        resolved_facts = []
+        scope6_interfaces = {}
+        interfaces = Facter::Resolvers::Aix::Networking.resolve(:interfaces)
+        primary = Facter::Resolvers::Aix::Networking.resolve(:primary)
+        interfaces&.each do |interface_name, info|
+          scope6_interfaces["scope6_#{interface_name}"] = info[:scope6] if info[:scope6]
+        end
+
+        scope6_interfaces.each { |fact, value| resolved_facts << Facter::ResolvedFact.new(fact, value, :legacy) }
```

Comment:
We could write this as
```ruby
        interfaces&.each do |interface_name, info|
          resolved_facts << Facter::ResolvedFact.new("scope6_#{interface_name}",  info[:scope6], :legacy) if info[:scope6]
        end
```

And drop `scope6_interfaces`

Reply:
Done